packages/prerender-proxy/package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Cloudfront Lambda@Edge constructs for integrating with prerender.io",
   "main": "index.js",
   "scripts": {
-    "build": "tsc && cd ./lib/handlers && npm ci",
-    "prepublish": "tsc && cd ./lib/handlers && npm ci"
```

Comment:
Not sure about this one... Do we not need to install in the handler directory?

Reply:
This is because we are no longer building it "outside" of CDK. When we're running Esbuild from within CDK, the packages are available from the construct's `package.json`

This let's CDK handle lambda functions as first class citizens instead of treating it as an asset that is built outside and bundled in cdk.